### PR TITLE
Add ability to add columns to sim output

### PR DESCRIPTION
**Pull request by Makar8000**

### CODE REVIEW

File: packages/frontend/src/scripts/sims/components/ability_used_table.ts [hunk 1]
```diff
@@ -28,7 +28,7 @@ function roundTime(time: number): string {
 
 export class AbilitiesUsedTable extends CustomTable<DisplayRecordFinalized> {
 
-    constructor(abilitiesUsed: readonly DisplayRecordFinalized[]) {
+    constructor(abilitiesUsed: readonly DisplayRecordFinalized[], extraColumns: CustomColumnSpec<DisplayRecordFinalized, any, any>[] = []) {
```
Comment (xpdota) [inline packages/frontend/src/scripts/sims/components/ability_used_table.ts:31]:
```suggestion
    constructor(abilitiesUsed: readonly DisplayRecordFinalized[], extraColumns: CustomColumnSpec<DisplayRecordFinalized, unknown, unknown>[] = []) {
```

**Comment by xpdota** [comment]
Looks fine apart from the lint issue

**Comment by Makar8000** [comment]
committed the suggested change to resolve

**Comment by xpdota** [comment]
Thank you!

**Pull request merged**